reports/templates/R/results_helper_functions.R

```diff
@@ -0,0 +1,165 @@
+#' 
+#' Takes a vector of filepaths to RData SuperLearner output from a P3 run
+#' and generates a list containing a few useful summary data structures.
+#'
+#' @author Keith Hughitt <khughitt@umd.edu>
+#'
+#' @param infiles Vector of post-processed P3 RData results files
+#' @return List of parsed SuperLearner learners, broken up into four parts:
+#          learner coefficients, cross-validation risks, random forest
+#          variable importance, and prediction variance.
+parse_p3_results <- function(infiles) {
+    # create placeholder variables
+    drugs              <- c()
+    coefs              <- NA
+    cv_risks           <- NA
+    feature_importance <- NA
+    prediction_var     <- c()
+
+    # iterate over P3 outputs
+    for (i in 1:length(infiles)) {
+        # load result
+        infile <- infiles[i]
+        message(sprintf("Loading %s.", infile))
+        load(infile)
+
+        # drug name
+        drugs <- c(drugs, p3_result$drug)
```

Comment:
@khughitt should be `c(drugs, p3_result$drug_name)` according to `tools/post_processing.R`?